refinery/analysis_manager/views.py

```diff
@@ -189,7 +189,7 @@ def run(request):
     analysis_status.save()
 
     # call function via analysis_manager
-    run_analysis.delay(analysis.uuid)
+    AnalysisRunner(analysis.uuid)
```

Comment:
The new way puts the logic in a constructor, but we don't actually have any need for the resulting object? Do we ever?